main.go

```diff
@@ -367,6 +370,54 @@ enabled: true`,
 			},
 		},
 	},
+	{
+		Name:        "render-template",
+		Usage:       "Render a Go template",
+		Description: "Render a Go template with machine state and config as data context",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "file",
+				Aliases:  []string{"f"},
+				Required: true,
+			},
+		},
+		Action: func(c *cli.Context) error {
```

Comment:
I think this is worth a test. Wrapping it in a function and moving the logic [here](https://github.com/kairos-io/kairos-agent/tree/main/pkg/action) will make it easier to test it. A simple one that checks that config values are actually rendered correctly in a template should suffice.

Reply:
Done. I have moved the logic into a separate action file and added test.